### PR TITLE
Added averageMedian function

### PR DESCRIPTION
**Pull request by qingwen23**
# Welcome to JavaScript community

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### **Describe your change:**

* Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
* This pull request is all my own work -- I have not plagiarized.
* I know that pull requests will not be merged if they fail the automated tests.
* This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* All new JavaScript files are placed inside an existing directory.
* All filenames should use the UpperCamelCase (PascalCase) style.  There should be no spaces in filenames.
     **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
* All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Maths/AverageMedian.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    }
+    
+    return median
+}
+
+function sortNumbers(num1, num2) {
+    return num1 - num2
+}
+
+console.log(averageMedian([8, 9, 1, 2, 5, 10, 11]))
```
Comment (raklaptudirm) [inline Maths/AverageMedian.js:39]: Please add the testing in a separate file in the `test` folder. We use `jest` for testing. You can check the files in `test` for help regarding how to write tests. The test file name should be `filename.test.js`.

**Review by raklaptudirm — CHANGES_REQUESTED** [review]

File: Maths/AverageMedian.js (hunk 1 above)
Comment (qingwen23) [reply Maths/AverageMedian.js]: Thanks for your review. I have added the test file in the new commit.

**Review by raklaptudirm — CHANGES_REQUESTED** [review]
The files need to follow the format specified by [`standard.js`](https://standardjs.com/). To format them, install the [`standard.js`](https://standardjs.com/) cli app and run `standard --fix .` in the repository. If it generates any warnings, fix them. This style allows our code to be readable and consistent.

**Review by raklaptudirm — APPROVED** [review]

**Comment by raklaptudirm** [comment]
@qingwen23 Thanks for contributing to the algorithms! We have repositories in many other languages as well if you are interested. You can also join our [discord server](https://discord.gg/c7MnfGFGa6). We are currently hosting a coding event there. Hope to see more contributions from you in the future!

**Comment by qingwen23** [comment]
@raklaptudirm Thanks! I would love to contribute more.

**Pull request merged**